content/concepts/nat/autonat.md

```diff
@@ -9,45 +9,79 @@ aliases:
 
 ## Background
 
-While the [identify protocol][spec_identify] lets peers inform each other about their observed network
-addresses, however, it is possible that some of these addresses are not accessible from outside the network, as the peer may be located in a private network that is behind a [NAT](overview.md) and therefore unreachable.
+While the [identify protocol](/concepts/introduction/protocols/identify)
+allows peers to inform each other about their observed network addresses,
+sometimes these addresses are unaccessible as the peer may be located in a
+private neatwork (i.e., behind a NAT or a firewall).
 
-To prevent this problem of advertising undialable addresses, libp2p has implemented a protocol called AutoNAT, which allows nodes to determine
-whether or not they are behind a NAT.
-Furthermore, this lays the foundation for nodes to find a way to improve their connectivity to peers in public networks.
+{{< alert icon="" context="info">}}
+Advertising addresses that are not reachable is detrimental for the health of a
+P2P network, as other nodes will unsuccessfully try to dial those addresses.
+{{< /alert >}}
+
+Nodes can use the AutoNAT protocol in libp2p to determine their network status and
+take appropriate action. For private nodes located behind a NAT, it is recommended
```

Comment:
```suggestion
To prevent this problem of advertising and dialing unreachable addresses, libp2p has implemented a protocol called AutoNAT, which allows nodes to determine whether or not they are behind a NAT.

For private nodes located behind a NAT, it is recommended:
```
Also is it recommended or required?

Reply:
strongly recommended